donate/payments/views.py

```diff
@@ -257,7 +258,7 @@ def process_monthly_transaction(self, form, send_data_to_basket=True):
                 form,
                 transaction_id=result.subscription.id,
                 last_4=payment_method.last_4,
-                send_data_to_basket=send_data_to_basket,
+                send_data_to_basket=False,
```

Comment:
@solarissmoke  I'm not sure if we should be doing this or not. When creating a monthly transaction that processes immediately, does the webhook still get fired? Or is this one of those things we just kind of need to find out with Braintree?

@stephaniemcv if someone makes a recurring donation (not an upsell), the first transaction should post immediately to basket, correct?

Reply:
The webhook still gets fired. At this stage all we have is a _subscription_, and no transactions have been created against it. The first transaction is not created immediately the subscription is created.